criterion.go

```diff
@@ -406,58 +412,3 @@ func criterionUnmarshalXML(dec *xml.Decoder, start xml.StartElement) (Criterion,
 		}
 	}
 }
-
-func criterionMarshalXML(c Criterion, e *xml.Encoder) error {
```

Comment:
I've removed it because anyway it was not working (i.e  it was adding a second time the  xsi:type)